components/call_button/call_button.tsx

```diff
@@ -50,56 +54,66 @@ function CallButton(props: Props) {
                     }
                 }}
             >
-                {item.icon}
-            </button>
+                {item.button}
+            </div>
         );
-    } else {
-        const pluginCallMethods = props.pluginCallMethods.map((item) => {
-            return (
-                <li
-                    key={item.id}
-                    onClick={(e) => {
-                        e.preventDefault();
-                        if (item.action) {
-                            item.action(props.currentChannel, props.channelMember);
-                        }
-                    }}
-                >
-                    <a href='#'>
-                        <span className='call-plugin-icon'>
-                            {item.icon}
-                        </span>
-                        {item.dropdownText}
-                    </a>
-                </li>
-            );
-        });
-        bodyAction = (
-            <MenuWrapper>
+    }
+
+    const pluginCallMethods = props.pluginCallMethods.map((item) => {
+        return (
+            <li
+                className='MenuItem'
+                key={item.id}
+                onClick={(e) => {
+                    e.preventDefault();
+                    if (item.action) {
+                        item.action(props.currentChannel, props.channelMember);
+                    }
+                }}
+            >
+
+                {item.dropdownButton}
+            </li>
+        );
+    });
+    return (
+        <div
+            style={style.container}
+            className='flex-child'
+        >
+            <MenuWrapper
+                onToggle={(toggle: boolean) => {
+                    setActive(toggle);
+                }}
+            >
                 <button
                     type='button'
-                    className='style--none post-action'
+                    className={'style--none call-button dropdown ' + (active ? 'active' : '')}
```

Comment:
Maybe a good use case for the `classnames` library?

Reply:
Clearly I didn't know about its existance, thanks :raised_hands: